app/components/Errors.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { map } from "lodash";
+import CSSModules from "react-css-modules";
+import styles from "../styles/components/errors.css";
+
+class Errors extends React.Component {
+  render () {
+    const { errors } = this.props;
+    const errorsMap = map(errors, (error, index) => {
```

Comment:
But it's not markup, it's just JS once it's compiled. That's the beauty in JSX hence why I think this kind of thing is OK. 

I think for simple loops like this, it's fine. Anything more in the child and you should break out to another function.